docs/general/ddl/data-types.rst

```diff
@@ -2867,6 +2862,45 @@ constant array values.
 
 .. _data-types-geo:
 
+Nested arrays
+..............
+
+Nested arrays cannot be used directly in column definitions  (i.e. 
+``ARRAY(ARRAY(DOUBLE))`` is not accepted), but multiple arrays can be nested
+as long as objects come between them (e.g. ``ARRAY(OBJECT AS (innerarray 
```

Comment:
That's just personal preference, I would put the whole expression in the next line.